.travis.yml

```diff
@@ -5,6 +5,19 @@ before_install:
 - docker build . --build-arg RAILS_ENV=test -t app/test
 jobs:
   include:
+  - stage: Terraform
+    name: validate and lint the Terraform
+    script: |
+      git clone https://github.com/tfutils/tfenv.git ~/.tfenv
+      export PATH="$HOME/.tfenv/bin:$PATH"
+      tfenv install
+      bash -c "$(curl -fsSL https://raw.github.com/cloudfoundry-community/terraform-provider-cf/master/bin/install.sh)"
+      terraform init terraform
+      terraform fmt  -diff -check -recursive terraform
+      terraform validate terraform
+  - stage: Shell
```

Comment:
Similar suggestion as https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/233#discussion_r387703624

Although this doesn't do any installing shell should change much less frequently than Ruby so it might be nice to add this check afterwards.